framework/src/main/java/org/checkerframework/common/wholeprograminference/WholeProgramInferenceScenes.java

```diff
@@ -31,20 +31,17 @@
 
 /**
  * WholeProgramInferenceScenes is an implementation of {@link
- * org.checkerframework.common.wholeprograminference.WholeProgramInference} that uses a helper class
- * ({@link org.checkerframework.common.wholeprograminference.WholeProgramInferenceScenesHelper})
- * that manipulates .jaif files to perform whole-program inference.
+ * org.checkerframework.common.wholeprograminference.WholeProgramInference} that uses a storage
+ * class ({@link
+ * org.checkerframework.common.wholeprograminference.WholeProgramInferenceScenesStorage}) that
+ * manipulates .jaif files to perform whole-program inference.
```

Comment:
The reference to `.jaif` files here should probably be to the `AScene` library instead, since that's what's actually stored. The scenes are only written out to `.jaif` files once, at the end.